src/payment/strategies/mollie/mollie-payment-strategy.ts

```diff
@@ -50,14 +59,19 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
         this._initializeOptions = mollie;
 
         const paymentMethods = state.paymentMethods;
-        const paymentMethod = paymentMethods.getPaymentMethodOrThrow(options.methodId);
+        const paymentMethod = paymentMethods.getPaymentMethodOrThrow(methodId, gatewayId);
         const { config: { merchantId, testMode } } = paymentMethod;
 
         if (!merchantId) {
             throw new InvalidArgumentError('Unable to initialize payment because "merchantId" argument is not provided.');
         }
 
         if (methodId === MolliePaymentMethodType.creditcard) {
+
+            if (mollie.form && this._isHostedPaymentFormEnabled(methodId, gatewayId) && this._isHostedFieldAvailable()) {
+                await this._mountCardVerificationfields(mollie.form);
+            }
```

Comment:
hey @JoseAlvarezHdz

This is a bit confusing. I was under the impression this PR was meant to replace the standard mollie form for a hosted field version. The screenshot shown in the PR description shows a saved credit card form, not the default one.

Also, in line https://github.com/bigcommerce/checkout-sdk-js/pull/1140/files#diff-ef86963fe7cb5db6143559804e7e2f0dc81a48ab8562e81b03ff1968b12f314aR76 we seem to be "mounting" the mollie form after we've already attached the hosted one.

I could be misreading this PR, but could you provide an screenshot where you show the actual hosted form being shown? I'm under the impression this code will try and render both, the hosted fields and mollie ones alongside each other.

Reply:
@icatalina 

This ticket is about **adding Verification Fields as Hosted Fields for TSV**, it does not REPLACE anything as Mollie didn't have verification Fields! 

This is how it looks with **Hosted Field for Verification** meaning the hosted fields were added for TSV not for cc form as Mollie **IS ALREADY HOSTED FORM**

![Screen Shot 2021-06-04 at 10 43 52](https://user-images.githubusercontent.com/69221626/120828117-c420ab00-c521-11eb-81e7-3933824e5a5a.png)

This is how it looks when adding a new CC when the client has a vaulted cc

![Screen Shot 2021-06-04 at 10 45 35](https://user-images.githubusercontent.com/69221626/120828334-06e28300-c522-11eb-9e4f-26d35d32b602.png)


And this is how it looks just as the CC FORM
![Screen Shot 2021-06-04 at 10 48 15](https://user-images.githubusercontent.com/69221626/120828645-617bdf00-c522-11eb-8c3d-08f02a807d2c.png)

Reply:
Yes, kind of. but i have made a change on it